File: DKS-API/DTOs/DtrFgtEtdDto.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DKS.API.Models.DKS
{
    public class DtrFgtEtdDto
    {
        public string FactoryId { get; set; }
        public string Article { get; set; }
        public string Stage { get; set; }
        public string Test { get; set; }
        public DateTime QcReceive { get; set; }

        public DateTime? QcEtd { get; set; }
        public string Remark { get; set; }
        public string? EtdUser { get; set; }
        public DateTime? EtdDate { get; set; }
        public string? LabNo { get; set; }
        
        public string? FgtUser { get; set; }
        public DateTime? FgtDate { get; set; }
    }
}

File: DKS-SPA/src/app/core/_models/dtr-fgt-etd-dto.ts
import { ModelInterface } from "./interface/model-interface";

export class DtrFgtEtdDto implements ModelInterface{
     factoryId :string;
     article :string;
     stage :string;
     test :string;
     qcReceive :Date;
     
     qcEtd :Date;
     remark :string;
     etdUser :string;
     etdDate :Date;
     labNo :string;

     fgtUser:string;
     fgtDate :Date;
}
